profiles/learner-profiles.md

```diff
@@ -1,5 +1,22 @@
 ---
-title: FIXME
+title: Learner Profiles
 ---
 
-This is a placeholder file. Please add content here.
+This page outlines example learner profiles. It is intended to highlight audiences for this Fortran course and provide example learning pathways into other courses. Instructors can use the profiles and pathways to decide which episodes to teach or place more emphasis on.
+
+## Profiles
+
+Before teaching a workshop consider who is attending and their needs.
+
+***Tahani is a new research software engineer at the Met Office.*** She has just joined the office after their degree and have experience in Python, Linux, and shell commands. She has limited experience with compiled languages. The weather and climate code she will be working on is written in Fortran. She thought Fortran was a dead language.
+
+***Ionas, a researcher joining the JULES (Joint UK Land Environment Simulator) partnership.*** They will be adding new functionality to JULES in Fortran. They have extensive experience with C and C++ and have been told there are differences between Fortran and C/C++ that they should be aware of. They are keen to learn Fortran but must fit their learning inbetween teaching duties and other research commitments.
+
+***Malak is mathematics graduate student working on new timestepping methods.*** Malaks previous work was mostly theoretical although he knows basic Python. Malak needs to use Fortran to prototype his new routines which will interface with other Fortran code. Malak has been told by his supervisor that he should set up his preferred text editor for Fortran development.
+
+***Joe has been employed by a tech company to update their Fortran code.*** Joe has years of experience developing code but has not used Fortran in a production environment. They have been asked to update legacy code to modern standards, including implementing some form of testing and using standard libraries. Joe is not sure if there is a standard code style that they can adopt.
```

Comment:
Is there value in splitting Joe into Experienced programmer in compiled language vs Experienced programmer in non-compiled language? As a Joe who came to fortran as an experienced programmer in non-compiled softly typed languages it was a double whammy of compiling and strict types that a c++ programmer would not face.

Reply:
I realise the C++ background variant is covered by Ionas and the python background (potentially at a more junior level) by Tahani